static/css/main.css

```diff
@@ -21,6 +21,7 @@ body {
     background: #f2f3f3;
     color: #000305;
     font-size: 87.5%; /* Base font size: 14px */
+/* TODO change the font */
```

Comment:
oh thank you! In reality I wanted to change the size, but it's just the line above :)
I still have to install this theme, I want it to be included in my repository.